src/main/java/com/ppkwu/ppkwu/services/SampleService.java

```diff
@@ -16,8 +16,32 @@ public String reverse(String stringToReverse, String reverseToString) {
     }
 
     public HashMap recognizeCharacter(String string_To_Recognize) {
-       HashMap hashMap = new HashMap();
-       return hashMap;
+        ArrayList Digits = new ArrayList();
+        ArrayList Letters1 = new ArrayList();
+        ArrayList Letters2 = new ArrayList();
+        ArrayList SIGNS = new ArrayList();
```

Comment:
We use camelCase naming convention within the project, so let's use it here as well.